meeting/meeting_june_9_2020.md

```diff
@@ -0,0 +1,40 @@
+# June 9th (7 PM CST)/10th (9 AM JST) Meeting 2020
+
+## Adaptor Signatures
+
+* We don't need the NIZK proof of knowledge of the scalar behind the adaptor point from the [generalized channel paper](https://eprint.iacr.org/2020/476.pdf), especially for DLCs
+  * The purpose of this proof is to maintain zero knowledge by ensuring that this scalar is already known and hence not learned.
+  * For DLCs, we assume knowledge of the adaptor secret (which is an oracle signature) will be made public and hence known/knowable to both parties
```

Comment:
```suggestion
  * The purpose of this proof is to make sure the adaptor point is well formed and not maliciously crafted. Since all our adaptor points come from an anticipated signature from the oracle's static key, we assume they are always well formed.
```